### DISCUSSION TITLE
How to exlude a page from keep-alive

### DISCUSSION THREAD

**Discussion by stenet** [nuxt/nuxt#16417] [Questions]
Hi,

I am trying to figure out how to exlude a page from keep-alive.

See [https://stackblitz.com/github/stenet/nuxt-keep-alive-demo?file=app.vue](https://stackblitz.com/github/stenet/nuxt-keep-alive-demo?file=app.vue). There is a link "No-keep-alive" (page = no-keep-alive.vue) that should be excluded.

What I have found out so far:

KeepAlive's direct child is an anonymous component (screenshot 2), which has the NoKeepAlive component. When KeepAlive is executed, the name of the component is undefined (screenshot 1).

Screenshot 1
![image](https://user-images.githubusercontent.com/1838511/191753123-5992032f-021e-49f9-8b53-8f609c6d2de5.png)

Screenshot 2
![image](https://user-images.githubusercontent.com/1838511/191753261-35a8f557-13b3-42a0-a42d-4e3f997ebc36.png)

Does anyone have any idea what I am doing wrong?
Thank you!

**Comment by bot08** [comment]
Try this: `<NuxtPage :keepalive="{ include: 'no-keep-alive' }" />` (page name)
Also this may be helpful: [#8367](https://github.com/nuxt/framework/issues/8367)

**Reply by stenet** [reply]
Thanks, but I couldn't get it to work with that either. I will follow #8367.
